src/pkrd/views/gen6/handler.rs

```diff
@@ -18,9 +20,16 @@ enum RightGen6View {
     DaycareView,
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+enum EntireGen6View {
```

Comment:
I agree with your PR comment about `BottomGen6View` reading better.  'Left', 'Right', and 'Bottom' implies left and right are for the top.  `Left', 'Right', and 'Entire' is a bit confusing.

Let's switch 'Entire' to 'Bottom'.

We can always rename the left and right screens to 'TopLeft' and 'TopRight' if it gets too confusing.